assets/agenda/components/AgendaCoverages.tsx

```diff
@@ -62,7 +63,7 @@ function AgendaCoveragesComponent({item, coverages, wireItems, actions, user, on
                         )}
                         key={coverage.coverage_id}
                         onClick={onClick}
-                        title={onClick ? gettext('Open {{agenda}} in a new tab', window.sectionNames) : onClick}
+                        title={gettext('Open {{agenda}} in a new tab', window.sectionNames) || onClick}
```

Comment:
I guess the logic was fine, just not sure about that ` : onClick` that imo should be rather `: ''` because there should be a string

Reply:
got it